service/templates/projects.html

```diff
@@ -391,23 +407,24 @@ <h2> Settings for [[displayName]]</h2>
 
           <div id="patch-aliases-list" class="form-group" ng-repeat="obj in patch_aliases track by $index">
             <div class="col-lg-2">
-              <input class="form-control" ng-model="patch_aliases[$index].alias" type="text" placeholder="alias">
+              <input class="form-control" ng-model="obj.alias" type="text" placeholder="alias">
             </div>
             <div class="col-lg-2">
-              <input class="form-control" ng-model="patch_aliases[$index].variant" type="text" placeholder="variant regex">
+              <input class="form-control" ng-model="obj.variant" type="text" placeholder="variant regex">
             </div>
             <div class="col-lg-2">
-              <input class="form-control" ng-model="patch_aliases[$index].task" type="text" placeholder="task regex">
+              <input class="form-control" ng-model="obj.task" type="text" placeholder="task regex">
             </div>
             <div class="col-lg-2">
-              <input class="form-control" ng-model="patch_aliases[$index].tags_temp" type="text" placeholder="tags (comma-delimited)">
+              <tag-input klass="form-control" items="obj.tags" />
```

Comment:
Aren't these defined as "tagInput"?

Reply:
AngularJS use "names normalization", which translates `camelCase` to `camel-case`